src/engine/controls/cuecontrol.cpp

```diff
@@ -1072,7 +1067,9 @@ void CueControl::hotcueActivatePreview(HotcueControl* pControl, double value) {
         const mixxx::audio::FramePos position = pControl->getPreviewingPosition();
         updateCurrentlyPreviewingIndex(Cue::kNoHotCue);
         m_pPlay->set(0.0);
-        seekExact(position.toEngineSamplePosMaybeInvalid());
+        if (position.isValid()) {
```

Comment:
Why did you remove this conditional? seekExact() doesn't seem to check for invalid positions.

Reply:
What do you mean? I added the valid check here because seeking to an invalid position doesn't make sense IMO.

Reply:
I misread the diff, confused :see_no_evil: